docker/Dockerfile

```diff
@@ -65,21 +49,39 @@ FROM base as builder
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Install Rust (specific version) and other build dependencies
-RUN apt-fast update && apt-fast install -y --no-install-recommends \
+RUN apt update && apt install -y --no-install-recommends \
     build-essential \
+    checkinstall \
```

Comment:
checkinstall has not been maintained for a few years. It broke for me this morning in CI, and I switched to `dpkg-deb` instead -- could you please use that package here too?

Reply:
TIL! Yep, I'll replace with dpkg-deb